web/CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+project(trweb)
+
+file(GLOB trweb_srcs "src/*.js" "style/*.scss")
```

Comment:
https://cmake.org/cmake/help/latest/command/file.html#glob
> **Note:** We do not recommend using `GLOB` to collect a list of source files from your source tree. If no CMakeLists.txt file changes when a source is added or removed then the generated build system cannot know when to ask CMake to regenerate. The `CONFIGURE_DEPENDS` flag may not work reliably on all generators, or if a new generator is added in the future that cannot support it, projects using it will be stuck. Even if `CONFIGURE_DEPENDS` works reliably, there is still a cost to perform the check on every rebuild.

Reply:
Good catch. Fixed in 500bf2ce277bf881ebfbbcf60a92c53bf35cd272